### DISCUSSION TITLE
Check-in app custom fields: is it possible?

### DISCUSSION THREAD

**Discussion by salvedgms** [alfio-event/alf.io#1414] [Q&A]
Hi everyone,

I'm a new happy Alf.io user and I have a question. Is it possible to add a custom field to sold tickets from the backoffice after the registration period has ended, but before the event begins, and have that field displayed on the mobile app during check-in?

Here’s the scenario: after the registration period, we have an event with about 100 registered attendees (sold tickets), and we’d like to divide them into 4 groups of 25 people each based on some a posteriori criteria determined in the backoffice.  During check-in, we want to quickly direct each attendee to their group.  While we could use a printed list to match names, it would be much more convenient if the group information appeared directly on the mobile app screen as soon as the QR code is scanned. This way, the team member using the mobile app would instantly know which group to direct the attendee to.

Thank you, best regards,
-- Davide

**Comment by cbellone** [comment]
Hi @salvedgms,
thank you for using alf.io! :+1: 

>  Is it possible to add a custom field to sold tickets from the backoffice after the registration period has ended, but before the event begins, and have that field displayed on the mobile app during check-in?

this is not possible at the moment. 
We currently display custom field values at check-in only if they are linked to an additional item purchased by the attendee. 
For example, we display "T-Shirt size" if the attendee purchased a "T-Shirt" additional item.

However, what you are suggesting seems a nice addition to our check-in functionality. 
I guess we could add a dedicated flag on the additional field (i.e. "display at check-in"), default: false.

Is this something you'd be willing to contribute or sponsor?

Thanks,
Celestino
